src/steamcompmgr.cpp

```diff
@@ -713,6 +717,69 @@ void MouseCursor::setDirty()
 	m_dirty = true;
 }
 
+bool MouseCursor::setCursorImage(char* data, int w, int h)
```

Comment:
I know gamescope's code style is terribly inconsistent, but we try to put the `*` on the right.